test/verify/check-networkmanager-firewall

```diff
@@ -138,12 +138,12 @@ class TestFirewall(NetworkCase):
 
         # Zones should be visible as unprivileged user
         self.login_and_go("/network/firewall", superuser=False)
-        self.wait_onoff("#firewall-heading-title-group", False)
+        self.wait_onoff("#firewall-heading-title-group", True)
```

Comment:
ouch! this was a race condition all the time, wasn't it? Thanks for spotting!

Reply:
Yes.  Putting the spinner in there exposed it.